src/PerformanceTests/Common/EnvironmentStats.cs

```diff
@@ -34,9 +33,9 @@ public static void Write()
         log.InfoFormat("UseNagleAlgorithm: {0}", ServicePointManager.UseNagleAlgorithm);
 
         log = LogManager.GetLogger("ENVVAR");
-        foreach (var item in Environment.GetEnvironmentVariables().OfType<DictionaryEntry>().OrderBy(x=>x.Key))
+        foreach (var key in Environment.GetEnvironmentVariables().Keys.OfType<string>().OrderBy(x=> x))
         {
-            log.DebugFormat("- {0}", item.Key);
+            log.InfoFormat("{0}: {1}", key, Environment.GetEnvironmentVariable(key));
```

Comment:
currently, it doesn't log the value for security reasons, so just using the string should be good enough it seems?